exercises/practice/sum-of-multiples/.meta/config.json

```
@@ -0,0 +1,18 @@
+{
+  "blurb": "Given a number, find the sum of all the multiples of particular numbers up to but not including that number.",
+  "authors": [],
```

@colinleach please add yourself as author here :)

```suggestion
  "authors": [
    "colinleach"
  ],
```